scripts/util/parse_path.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+
+def create_path_from_format_string(
```

Comment:
move this to cares_rl - helpers.py, generally useful for other training environments.